pkg/internalapis/cnsvolumeoperationrequest/v1alpha1/cnsvolumeoperationrequest_types.go

```diff
@@ -42,10 +41,9 @@ type CnsVolumeOperationRequestStatus struct {
 	// ErrorCount is the number of times this operation failed for this volume.
 	// Incremented by clients when new OperationDetails are added with error set.
 	ErrorCount int `json:"errorCount,omitempty"`
-	// Reserved keeps a track of the quantity that should be reserved in
-	// storage quota during a create/expand volume operation.
-	// +optional
-	Reserved *resource.Quantity `json:"reserved,omitempty"`
+	// StorageQuotaDetails stores the details required by the CSI driver and syncer to
```

Comment:
This should optional and it should be a pointer.

Reply:
It is already marked as omitempty, so it is optional. Why does it need to be a pointer though?

Reply:
If it is not a pointer, it will still be an empty struct. If it is a pointer, it is nil if not set.
You should add "+optional" in the comment to indicate it is optional.

Reply:
Good catch, updated it